### ISSUE TITLE
update ActiveMQ routes for argo-stage

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/argo#160]
when we deploy to stage, we need to update the ActiveMQ routing to go to the new argo VM(s) for the reindexing pipeline

**Comment by drh-stanford** [comment]
created SAR-3010 to delete karaf from fedora VMs as we're no longer using it for our message routing...

**Comment by atz** [comment]
May want to stall or retract that ticket and karaf is still used during bulk reindexing, according to @cbeer.

**Comment by atz** [comment]
Followed up w/ @cbeer and he confirms the deletion is OK.  Local instances are unused and unintended.

**Comment by drh-stanford** [comment]
@eefahy Hi Erin, when we hook up argo stage with DOR test we will need to reroute the sul broker to go to the new stage environment. thanks.

**Comment by jmartin-sul** [comment]
just discussed w/ @eefahy and @cbeer.  they said that while it's possible to send reindexing messages to both argo-test and argo-stage-b, the setup is non-trivial, and they were wondering if it was necessary yet.

my understanding of stage and test is that people will continue to use argo-test to simulate current production usage (and possibly to prototype objects that will later be created on prod?  i think ahmed used to do this?).  simultaneously, the argo development team will use argo-stage to test indexing performance and the mechanics of managing full reindexing (or reindexing of large known batches of objects).  it's also my understanding that message queuing supports automatic reindexing upon saving an object in fedora, or batch reindexing by way of handing druid lists to karaf.

since argo-stage's web UI isn't being used by a wider audience yet, and since we're first using stage to explore options other than the karaf bulk reindexing mechanism, it seemed that we could defer for now on setting up message routing to argo-stage.

it is possible that there will be a period in which argo-test and argo-stage will coexist as web interfaces, with the expectation that things like the indexing queue will work as usual on both simultaneously.  but it also seems possible that we'll just cut over to stage at some point, which chris and erin say is much less work than sending messages to both.  so, we figured we could wait and see whether we landed in the simultaneous web UI situation or the cut-over situation before doing any possibly unnecessary work.

also possible that i missed or misunderstood some use case for the message queuing/routing and that things still need to be routed to both, or rerouted, despite the above.
